kernel_tuner/convert/convert.py

```diff
@@ -0,0 +1,102 @@
+import json
+from pathlib import Path
+
+# The path construction here probably needs to be changed when this file becomes part of the library
+PROJECT_DIR = Path(__file__).parents[0]
+
+SCHEMA_VERSIONS_PATH = PROJECT_DIR / "../schema/cache/convert_temp"
+
+DEFAULT_VALUES = {
+    "object":   dict(),
+    "array":    list(),
+    "string":   "default string",
+    "integer":  0,
+    "number":   0,
+    "true":     True,
+    "false" :   False,
+    "null":     None
+}
+
+
+def convert_cache_file(filestr):
```

Comment:
Ik zou deze functie splitten in een functie die een cache _file_ omzet en een functie die een cache _object_ (JSON) omzet